subsys/net/lib/nrf_cloud/coap/cddl/nrf_cloud_coap_agps.cddl

```diff
@@ -8,16 +8,14 @@
 ; AGPS Request
 ; ************
 
-type = rtAssistance/custom
-
 agps_req = {
-	? customTypes => [1*10 int],
+	? customTypes => tstr,
 	eci => uint,
 	? filtered => bool,
 	? mask => uint,
 	mcc => uint,
 	mnc => uint,
-	? requestType => type,
+	? requestType => tstr,
```

Comment:
The `requestType` is already optional, and on the backend, it is deprecated anyway, since the backend can infer the type based on the other fields.  So there is no need to optimize this -- I will likely not send this field.

Reply:
Could we remove then this field?

Reply:
yes